### ISSUE TITLE
508 Defect Level 1, screen readers, checkbox to opt in and out of text alerts, status doesn't announce when changed

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#23532]
Label: 15247

    - [508-defect-1]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1

Issue Title: 508 Defect Level 1, screen readers, checkbox to opt in and out of text alerts, status doesn't announce when changed.

Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a user of screen reading technology, when I check or uncheck a checkbox I expect the screen reader to announce the new status. 

<h2>Details</h2>
When I check or uncheck the checkbox to opt into or out of text alerts, my screen reader does not announce the new status. The page refreshes and focus is placed at the top, causing me to have to navigate back to the checkbox to check its status.

<h2>Acceptance Criteria</h2>
 - [ ] Screen readers announce status of checkbox when checked or unchecked.

Environment
<ul>
<li>JAWS and Chrome,</li>
<li>NVDA and Chrome,  </li>
<li>JAWS and Firefox</li>
<li>NVDA and Firefox</li>
</ul>
Steps to Recreate
Sign into Profile 2.0 as User36. The checkbox is located on the Personal and Contact Information page where the user is given the option to update their mobile number.

**Comment by joshkimux** [comment]
This ticket is a sister to https://github.com/department-of-veterans-affairs/va.gov-team/issues/23168

**Comment by erikphansen** [comment]
A proper fix for this would require rethinking the UX for this checkbox that immediately kicks off an API request when its state changes and would likely require major development effort to change how it works.

So given that, I think the best approach would be to remove the checkbox from this "read" view of the Mobile Phone Number. The checkbox could be replaced with some supporting text explaining if they are currently getting health care appointment reminders or not. If they want to change that setting, they would enter the "edit" view.

**Comment by ajakabcin** [comment]
Designs for the proposed solution can be tracked in #24572

**Comment by ajakabcin** [comment]
@erikphansen final copy and designs for solution linked [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/24572#issuecomment-840642601)

**Comment by erikphansen** [comment]
@AngelaFowler82  We removed the checkbox and the new version should be on staging soon. Can you reevaluate how the mobile phone number works? Thank you.

**Comment by AngelaFowler82** [comment]
Absolutely. Can you let me know when it’s on staging?

**Comment by erikphansen** [comment]
@AngelaFowler82 it's there 🎉

**Comment by AngelaFowler82** [comment]
Looks good to me. The option to edit notifications works as expected.

**Comment by joshkimux** [comment]
Closing! 🥳
